File: OtherExercises/TasksInJavaScript/Implementation/sumMultiplesOfThreeAndFive.js
function sumMultiplesOfThreeAndFive(limit) {
    let sum = 0;
    for (let number = 1; number < limit; number++) {
        if (number % 3 === 0 || number % 5 === 0) {
            sum += number;
        }
    }
    return sum;
}

module.exports = sumMultiplesOfThreeAndFive;

File: OtherExercises/TasksInJava/Implementation/SumMultiplesOfThreeAndFive.java
public class SumMultiplesOfThreeAndFive {
    public static int sumMultiplesOfThreeAndFive(int upperLimit) {
        int sum = 0;
        for (int number = 1; number < upperLimit; number++) {
            if (number % 3 == 0 || number % 5 == 0) {
                sum += number;
            }
        }
        return sum;
    }
}
